<launch>
    <node pkg="ipc" type="ipc_node" name="ipc_node" output="screen">
        <!-- publish topic -->
        <remap from="cmd"          to="/planning/pos_cmd"/>
        <remap from="astar_path"   to="/astar/path"/>
        <remap from="grid_map"     to="/astar/grid_map"/>
        <remap from="sfc"          to="/mkr_arr"/>
        <remap from="mpc_path"     to="/mpc/path"/>
        <remap from="px4ctrl"      to="/mavros/setpoint_raw/attitude"/>
        <remap from="goal_pub"     to="/mpc/goal"/>
        
        <!-- subscribe topic -->
        <remap from="local_pc"     to="/cloud_registered"/>
        <remap from="odom"         to="/mavros/local_position/odom"/>
        <remap from="imu"          to="/livox/imu"/>
        <remap from="goal"         to="/goal"/>
        <remap from="rc_in"        to="/mavros/rc/in"/>

        <rosparam command="load" file="$(find ipc)/config/param_gazebo.yaml"/>

    </node>
    <node name="ipc" pkg="rviz" type="rviz" args="-d $(find ipc)/config/env_real.rviz"/>
</launch>
